extensions/Worker.Extensions.Storage.Blobs/src/BlobStorageConverter.cs

```diff
@@ -60,6 +60,14 @@ private async ValueTask<ConversionResult> ConvertFromBindingDataAsync(ConverterC
                 }
 
                 BlobBindingData blobData = GetBindingDataContent(modelBindingData);
+
+                // Temp fix for connection value being incorrect from WebJobs
+                context.TryGetBindingAttribute<BlobTriggerAttribute>(out object attribute);
+                if (attribute is BlobTriggerAttribute blobTriggerAttribute)
+                {
+                    blobData.Connection = blobTriggerAttribute.Connection;
```

Comment:
There is a [PR](https://github.com/Azure/azure-sdk-for-net/pull/38284) in the WebJobs ext to find this. We can also fix this issue here if necessary.